.halcyon/cabal-version

```diff
@@ -0,0 +1 @@
+1.22.4.0
```

Comment:
Can the GHC version be put into a file? Right now I have it in the config setting, which is less than ideal.